### ISSUE TITLE
Rephrase the statement "Complete support of Go specification" ?

### ISSUE THREAD

**Issue by cosmos72** [containous/yaegi#287]
First of all, congratulations for successfully writing a Go interpreter, and for the popularity that yaegi is quickly gaining :)

To be completely honest, I was baffled when reading the sentence "Complete support of Go specification". I thought:
"how is it possible using only the standard library? I had to write a lot of workarounds and emulations in my interpreter, and still it's not 100% compliant with Go specification."

As you certainly realized, there are a lot of corners in Go specifications that are **hard** to fully implement in a Go interpreter that uses only Go standard library. At least the following ones come to my mind:
* declaring new interface types - `reflect` cannot do it
* add methods to types in order to implement interfaces - again, `reflect` cannot do it
* embedded fields - tricky to implement correctly, and `reflect` almost never generates wrapper methods
* import, especially for third party packages (i.e. outside Go standard library)
* differences between named types and their underlying type

There are already several open issues on these topics: #251 #257 #264 #270 #281

Now, for sure some of the issues are simply bugs. But others are features not implemented yet - stuff related to interfaces, for example.

For these reasons, I find it quite misleading that the README.md prominently states
```
Complete support of Go specification
```
Would you please be so honest to rephrase it to more accurately state the current situation?
Just to give an example, something like
```
Support most of Go specification
```
would surely be more accurate.

Disclaimer: I am the author of "yet another" Go interpreter.

**Comment by mvertes** [comment]
Thanks for your feedback @cosmos72, and congrats for your project, [gomacro](https://github.com/cosmos72/gomacro), which is really amazing.

You are raising valid points, we need to clarify the doc.

We say we support the complete [Go specification](https://golang.org/ref/spec), which relates to the language syntax and semantics of Go in a generic way. Yaegi implements (with some bugs, as you noticed) *all* of the features described in the spec. Interfaces, methods, hidden struct fields, defer, import, etc. All of it. See [_test/](https://github.com/containous/yaegi/tree/master/_test) to get an idea of what we test for.

But the spec should not be confused with the details of a particular platform, which is out of scope and not covered.  For yaegi to be more useful, we let the interpreter import packages in a pre-compiled binary form from the host executable, instead of reading go source files. That defines a new platform which needs to be described, in its capacities and limitations.

This reference is missing right now, we're working on it. In the mean time, and to remove confusion, what I propose is to add a **Limitations** section in the readme. 

What do you think ?

**Comment by cosmos72** [comment]
Yes, a **Limitations** section makes sense and would also guide users to understand if yaegi is suitable for their use cases.
